OpenTrons/opentrons-api
# feat(protocol-designer): add the release notes button to settings

re AUTH-856



# Overview



Add a `View release notes` button to the Settings page, and display the `AnnouncementModal` when the button is clicked.

## Test Plan and Hands on Testing


- Click on the settings icon
- Ensure it matches the [design](https://www.figma.com/design/WbkiUyU8VhtKz0JSuIFA45/Feature%3A-Protocol-Designer-Phase-1?node-id=3150-116483&node-type=canvas&t=VjSlf85g2lJzT7Ed-0)
- Verify that the Announcement modal appears when the 'View release notes' button is clicked.

## Changelog



- Added a button labeled 'View release notes' next to the PD version with a 12px gap
- Create an `AnnouncementModalProp` interface with optional `isViewReleaseNotes` and `onClose` props to control the visibility of the AnnouncementModal, tracking when the user clicks the button and closes the modal
- Used `useLocation` to check if the user is on the Settings page
- Updated tests 
## Review requests



## Risk assessment
